faasmcli/faasmcli/tasks/dev.py

```diff
@@ -65,8 +65,8 @@ def tools(ctx, clean=False, build="Debug", parallel=0, sanitiser="None"):
     """
     Builds all the targets commonly used for development
     """
-    nosgx = sanitiser != "None"
-    cmake(ctx, clean=clean, build=build, sanitiser=sanitiser, nosgx=nosgx)
+    sgx = "Disabled" if sanitiser != "None" else "Simulation"
+    cmake(ctx, clean=clean, build=build, sanitiser=sanitiser, sgx=sgx)
```

Comment:
Several magic strings in this function now, would be nice to put them as constants at the top of the file, e.g.

```
SGX_MODE_SIM = "Simulation"
SGX_MODE_DISABLED = "Disabled"
SANITISER_NONE = "None"
```